uniswap-v2-core/test/UniswapV2Pair.spec.ts

```diff
@@ -440,31 +440,36 @@ describe('UniswapV2Pair', () => {
     expect(await token1.balanceOf(pair.address), "Token 1 balance of pair").to.eq(bigNumberify(1000).add(token1ExpBalVexchange))
   })
 
-
   /**
    *  recoverToken - error handling for invalid tokens 
    */
   it('recoverToken:invalidToken', async () => {
+    let recoveryAddress = other.address
```

Comment:
Prefer `const` when not re-assigning. This would usually be caught by a linter but im unsure if Uniswap have that in the repo

Reply:
okay I changed all appropriate instances to use `const` instead of `let`